dev_tools/notebooks/isolated_notebook_test.py

```diff
@@ -43,6 +43,16 @@
     'docs/noise.ipynb',
     # Named topologies
     'docs/named_topologies.ipynb',
+    # get_qcs_objects_for_notebook
+    'docs/tutorials/google/calibration_api.ipynb',
+    'docs/tutorials/google/colab.ipynb',
+    'docs/tutorials/google/echoes.ipynb',
+    'docs/tutorials/google/floquet_calibration_example.ipynb',
+    'docs/tutorials/google/reservations.ipynb',
+    'docs/tutorials/google/spin_echoes.ipynb',
+    'docs/tutorials/google/start.ipynb',
+    'docs/tutorials/google/visualizing_calibration_metrics.ipynb',
+    'docs/tutorials/google/xeb_calibration_example.ipynb',
```

Comment:
Right below this line, all of the google notebooks are skipped anyway. Do we need these duplicates ?

Reply:
Yes, please see the comment below or the PR description for more details.